tsl/src/chunk_copy.c

```diff
@@ -1206,6 +1248,8 @@ chunk_copy_cleanup_internal(ChunkCopy *cc, int stage_idx)
 		else
 			first = false;
 
+		if (BOOTSTRAP_SUPERUSERID != saved_uid)
+			SetUserIdAndSecContext(saved_uid, sec_ctx);
```

Comment:
Again, this new code path is never tested.

Reply:
Added test coverage